Makefile

```diff
@@ -77,6 +77,14 @@ help: ## Display this help.
 
 ##@ Development
 
+go-version:
+	GO_MINOR_VERSION = $(shell $(GO) version | cut -c 14- | cut -d' ' -f1 | cut -d'.' -f2)
+	MINIMUM_SUPPORTED_MINOR_VERSION = 17
+	@if [ $(GO_MINOR_VERSION) -lt $(MINIMUM_SUPPORTED_MINOR_VERSION) ]; then \
+		echo 'Golang version is not supported, please update.';\
+		exit 1; \
+	fi
+
```

Comment:
I tried the PR and the command does not work. Also 2 general comments:
1. Please check major version as well otherwise 0.17 could pass the check.
2. In error message please specify the minimal version required.

```
$ make go-version
bash: line 1: version: command not found
GO_MINOR_VERSION = 
bash: line 1: GO_MINOR_VERSION: command not found
make: *** [Makefile:80: go-version] Error 127

```